### PR TITLE
Allow all logging policies

### PR DESCRIPTION
**Pull request by asmacdo**
@aaronkanzer and I made this change for LINC, and it was necessary to bring down BICAN. I've already made the changes for BICAN  and Mcgovern.

The file changed here is for reference only, it is not connected to automation.

### CODE REVIEW

File: .aws/terraform-jupyterhub-provisioning-policies.json [hunk 1]
```diff
@@ -169,14 +169,7 @@
...
         "kms:UpdateAlias",
-        "logs:CreateLogGroup",
-        "logs:DeleteLogGroup",
-        "logs:DeleteRetentionPolicy",
-        "logs:DescribeLogGroups",
-        "logs:ListTagsLogGroup",
-        "logs:PutRetentionPolicy",
-        "logs:TagResource",
-        "logs:UntagResource",
+        "logs:*",
```
Comment (yarikoptic) [inline .aws/terraform-jupyterhub-provisioning-policies.json:172]: where do those logs go?

**Comment by asmacdo** [comment]
The terraform automatically creates log groups in CloudWatch, but FWIW the logs being collected have not been useful to me yet.

Screenshot from 2024-08-12 12-13-55

Screenshot from 2024-08-12 12-14-09

**Comment by yarikoptic** [comment]
how to establish their archival?  we already use `heroku` cli on drogon to get the ones from heroku, and some from papertrail, and we use con/tinuous for stuff from github and other CIs.  I would imagine such logs could come handy for various debugging purposes.

**Comment by asmacdo** [comment]
@yarikoptic they are already archived within CloudWatch. Feel free to file an issue if you'd like them to be synced elsewhere, but I think it would require a security audit-- we don't want to be copying secrets around.

**Comment by asmacdo** [comment]
@aaronkanzer and I made this change together, its live for LINC and dandihub, so I feel comfortable merging.

**Pull request merged**